### PR TITLE
adds handholding (again???)

### PR DESCRIPTION
**Pull request by Detective-Google**
## About The Pull Request
Allows you to hold hands with other players by targeting their arms when grabbing 

## Why It's Good For The Game
being able to rp realistic human relationships is a good addition and also orange man said I could (even if he doesn't merge it /stealth reverts like last time we got hand holding 😢 )
image

## Changelog
:cl:
add: Coworkers are now encouraged to show affection [WITHIN LEGAL LIMITS] by holding eachother's hands!
/:cl:

### CODE REVIEW

**Comment by Anonmare** [comment]
Degenerates like you belong on the cross

**Comment by Moonman121** [comment]
Fetish content

**Comment by stylemistake** [comment]
> show affection

It makes total sense to politely grab a hand (instead of a full body grab) and drag someone to a different department, and this is the most common case of non-aggressive grab. I feel this is the most common case of a passive grab, and politeness is a huge factor in RP.

You need to chill your ERP boners.

**Comment by FreshestSoup** [comment]
when do we get ass grabbing

**Comment by waterwater1212** [comment]
what the fuck are you thinking

**Comment by TheRespectedMan** [comment]
we really do live in a society

**Comment by Shadowflame909** [comment]
I'll only accept this PR if nonconsensual hand-holding becomes a Capital offense via space-law,

**Comment by Detective-Google** [comment]
> I'll only accept this PR if nonconsensual hand-holding becomes a Capital offense via space-law,
> 
> Edit: Might actually give the lawyer something to do.

Add it to the wiki

Also when is the last time anyone saw shadowflame actually play kek

**Comment by 81Denton** [comment]
What if we held hands... in dorms 4 👀

**Comment by Shadowflame909** [comment]
Wouldn't you like to know Google?

Not selling my data that easily.

File: code/modules/mob/living/living.dm [hunk 1]
```diff
@@ -277,9 +277,9 @@
 
 		log_combat(src, M, "grabbed", addition="passive grab")
 		if(!supress_message && !(iscarbon(AM) && HAS_TRAIT(src, TRAIT_STRONG_GRABBER)))
-			M.visible_message("<span class='warning'>[src] grabs [M] passively!</span>", \
-							"<span class='warning'>[src] grabs you passively!</span>", null, null, src)
-			to_chat(src, "<span class='notice'>You grab [M] passively!</span>")
+			M.visible_message("<span class='warning'>[src] grabs [M] [(zone_selected == "l_arm" || zone_selected == "r_arm")? " by their hands!":"passively!"]</span>", \
```
Comment (Ghommie) [inline code/modules/mob/living/living.dm]:
```suggestion
			M.visible_message("<span class='warning'>[src] grabs [M] [(zone_selected == "l_arm" || zone_selected == "r_arm")? "by their hands":"passively"]!</span>", \
```

File: code/modules/mob/living/living.dm [hunk 2]
```diff
@@ -277,9 +277,9 @@
...
+							"<span class='warning'>[src] grabs you [(zone_selected == "l_arm" || zone_selected == "r_arm")? " by your hands!":"passively!"]</span>", null, null, src)
```
Comment (Ghommie) [inline code/modules/mob/living/living.dm]:
```suggestion
							"<span class='warning'>[src] grabs you [(zone_selected == "l_arm" || zone_selected == "r_arm")? "by your hands":"passively"]!</span>", null, null, src)
```

File: code/modules/mob/living/living.dm [hunk 3]
```diff
@@ -277,9 +277,9 @@
...
+			to_chat(src, "<span class='notice'>You grab [M] [(zone_selected == "l_arm" || zone_selected == "r_arm")? "by their hands!":"passively!"]</span>")
```
Comment (Ghommie) [inline code/modules/mob/living/living.dm]:
```suggestion
			to_chat(src, "<span class='notice'>You grab [M] [(zone_selected == "l_arm" || zone_selected == "r_arm")? "by their hands":"passively"]!</span>")
```

**Review by Ghommie — COMMENTED** [review]
Kneejerk suggestions on double spacing and exclamation marks.

**Comment by Tharcoonvagh** [comment]
Small step for a man
A giant leap for ERP

**Comment by Tharcoonvagh** [comment]
Small step for a man
A huge leap for ERP

File: code/modules/mob/living/living.dm (hunk 1 above)
Comment (stylemistake) [reply code/modules/mob/living/living.dm]: please don't suggest single-lining when humans don't read horizontally into infinity.

Comment (Ghommie) [reply code/modules/mob/living/living.dm]: I'm not suggesting that now, am I?

Comment (stylemistake) [reply code/modules/mob/living/living.dm]: Oh, right, PR author suggests that.

**Comment by MCterra10** [comment]
Inb4 oranges merges this and immediately commits a revert

**Comment by Iatots** [comment]
> 
> > show affection
> 
> It makes total sense to politely grab a hand (instead of a full body grab) and drag someone to a different department, and this is the most common case of a passive grab, and politeness is a huge factor in RP.
> 
> You need to chill your ERP boners.

No coworker grabs your hand IRL to lead you anywhere unless you are a child and your coworker is your teacher.

**Comment by MMMiracles** [comment]
well considering how most of the crew acts when they don't get what they want that seems like a reasonable assumption

**Comment by nicbn** [comment]
Sup reddit

**Comment by Farquaar** [comment]
Are people able to use whatever's in their hand if their hand is grabbed?
I'm not against it in theory, but kinetically it doesn't make sense that you're holding someone's hand and they're still able to interact with the environment with it.

**Comment by EightBitrous** [comment]
I’m fucking sorry?

**Comment by GuyonBroadway** [comment]
Add fun-buzzers so if anyone tries to handhold you they get zapped.

Also add hand-beartraps.

**Comment by HomocidlePancakes** [comment]
Hello, may name is gay pancakes and I want a special interaction with grab for every body part.

**Comment by Oshibkka** [comment]
why the actual fuck do you think this is a good idea

**Comment by Timonkeyn** [comment]
Hand and Sweat fetish content

**Comment by PKPenguin321** [comment]
It's all so tiresome

**Comment by Couls** [comment]
what if we held hands on the escape shuttle?

just kidding.. unless?

**Comment by mrhugo13** [comment]
asfasg

**Comment by FancyCyclops** [comment]
h-hold my hand

**Comment by DerpehBoi** [comment]
owo whats dis

**Pull request merged**